client/src/components/Configure.tsx

```diff
@@ -228,7 +232,7 @@ const Configure: React.FC = () => {
               <button
                 className={`btn btn-success-outline ${startLoading ? "hover:!text-[#d6dbdf] hover:!border-[#d6dbdf]" : ""}`}
                 onClick={handleSubmit}
-                disabled={startLoading}
+                disabled={startLoading || isRoundTimeInvalid()}
```

Comment:
Could you add a message to also correct the fields, if any of them are red, in the modal?
It's not clear to the user that something is wrong just because It's red.
Maybe a message like "Fix the errors above to continue"